packages/ember-routing/lib/system/router.js

```diff
@@ -542,6 +553,8 @@ var EmberRouter = EmberObject.extend(Evented, {
       emberRouter.didTransition(infos);
     };
 
+    // TODO
+    //router.willTransition = this.willTransition.bind(this);
```

Comment:
Hm, I think this was us saying we should just replace the line below with a `bind` since that's exactly what it was doing, but maybe we didn't want to change unrelated code in the commit